tools/helpers.cmake

```diff
@@ -396,9 +390,9 @@ function(config_string optionname configname doc)
         if(CONFIG_UNQUOTE)
             set(quote "")
         else()
-            set(quote "\"")
+            set(quote "\\\\\\\\\\\\\"")
```

Comment:
Can you add a comment explaining the escaping calculation that's going on here?

Reply:
Second to that, I had a similar thought counting the slashes.

Reply:
I came up with a better solution that involves less fighting with CMake:

https://github.com/seL4/seL4/pull/975/files#diff-67838857cc14e477be77851c70d3611d8c215fe0861678ec948e1b0859aade77R393
https://github.com/seL4/seL4/pull/975/files#diff-67838857cc14e477be77851c70d3611d8c215fe0861678ec948e1b0859aade77R393

Reply:
Is the second URL supposed to be to a different change line?

Reply:
Oops, thanks! Now it points to the correct line.

Reply:
Thanks, now I understand what the change is accomplishing!